x/ibc/04-channel/types/channel.go

```diff
@@ -13,6 +13,8 @@ var (
 	_ exported.CounterpartyI = (*Counterparty)(nil)
 )
 
+const DefaultChannelVersion = "1.0.0"
```

Comment:
Would appreciate a comment on this one (I personally have no idea what this version means, and when it's updated)

Reply:
this string is kinda just a placeholder for a feature that hasn't been implemented yet, I'm implementing it in #6640 and will give it a comprehensive description :+1: